smsgateway/utils.py

```diff
@@ -20,7 +22,15 @@ def check_cell_phone_number(number):
     return format_number(parsed_number, PhoneNumberFormat.E164)
 
 
-def truncate_sms(text, max_length=160):
+def get_max_msg_length():
+    account = get_account()
+    try:
```

Comment:
Yep, good point 👍 The pattern I used is quite common throughout our code base but not having to rely on an exception and having to write less code seems a win win. Will amend.